src/Form/InputCheckbox.php

```
@@ -0,0 +1,271 @@
+        }
+
+        $attributes['checked'] = false;
+
+        foreach ($normalizedChecked as $item) {
+            if (is_scalar($item) && "{$item}" === $valueStr) {
+                $attributes['checked'] = true;
+            }
+        }
+
+        return $attributes;
+    }
```

PHPMD (2.15.0)

[warning] The method buildAttributes() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)